Wallet -> Colelctibles: application crash (hard to reproduce)

### Description

I have a bunch of w/o accounts added so really hard to tell which one is having the collectible we don't handle correctly, but sometimes i see the crash with the following stacktrace:

```
Traceback (most recent call last)
/Users/anastasiya/status-desktop/src/nim_status_client.nim(193) nim_status_client
/Users/anastasiya/status-desktop/src/nim_status_client.nim(190) mainProc
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(558) onSlotCalled
/Users/anastasiya/status-desktop/src/app/core/signals/signals_manager.nim(70) receiveSignal
/Users/anastasiya/status-desktop/src/app/core/signals/signals_manager.nim(37) processSignal
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(1035) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(951) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(900) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(900) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(911) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(900) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(911) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(901) parseJson
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/json.nim(392) []=
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/collections/tableimpl.nim(52) []=
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/collections/tableimpl.nim(22) rawInsert
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/system/gc.nim(508) newObjRC1
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/system/alloc.nim(735) rawAlloc
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Segmentation fault: 11
```

@IvanBelyakoff also was able to reproduce it from time to time

--- a/services/wallet/collectibles/collectibles.go
+++ b/services/wallet/collectibles/collectibles.go
@@ -21,6 +21,8 @@ const requestTimeout = 5 * time.Second
 
 const hystrixContractOwnershipClientName = "contractOwnershipClient"
 
+const maxNFTDescriptionLength = 1024
+
 type Manager struct {
 	rpcClient                         *rpc.Client
 	mainContractOwnershipProvider     thirdparty.NFTContractOwnershipProvider
@@ -270,6 +272,12 @@ func (o *Manager) processAssets(chainID uint64, assets []opensea.Asset) error {
 					assets[idx].ImageURL = metadata.ImageURL
 				}
 			}
+		}
+
+		// The NFT description field could be arbitrarily large, causing memory management issues upstream.
+		// Trim it to a reasonable length here.
+		if len(assets[idx].Description) > maxNFTDescriptionLength {
+			assets[idx].Description = assets[idx].Description[:maxNFTDescriptionLength]
 		}
 
 		o.nftCache[chainID][id.HashKey()] = assets[idx]

